scalableminds/webknossos
# Upload volume tasks

### URL of deployed dev instance (used for testing):
- https://uploadvolumetasks.webknossos.xyz

### Steps to test:
- Upload a single zip containing multiple volume zips (e.g. project download)
- Should create as many tasks as included volume zips

### Issues:
- fixes #4401 

------
